### PR TITLE
mouse position display

### PR DESCRIPTION
**Pull request by Kethku**
Adds the mouse pixel position to the stats display for debugging purposes.

image

This is useful for locating exactly where you want a sprite to go on the screen. Instead of fiddling with the values, you can place your mouse where you want it and use the reported values.

### CODE REVIEW

**Comment by Kethku** [comment]
This does not take into account the current camera position, and will always output the mouse position in pixel coordinates. Up to you which would be more useful, or even if you want to merge this, I just found it useful when tracking down the getPixel bug.

**Comment by gabrielflorit** [comment]
I do see the value in this - but somehow I feel like it's not the right thing for SCRIPT-8. It's just a feeling though, and I may be wrong (this happens a lot!). Closing for now.

File: src/iframe/src/Iframe.js [hunk 1]
```diff
@@ -933,12 +944,16 @@ class Iframe extends Component {
...
-              className={classNames('fps', {
-                hide: isPaused
-              })}
+            <div className={classNames('stat-values', {
+              hide: isPaused
+            })}
             >
-              fps (avg): <span>{fps}</span>
+              <div className={classNames('stat')}>
+                mouse: (<span>{this.state.mousePosition.x}</span>, <span>{this.state.mousePosition.y}</span>)
```
Comment (icarito) [inline src/iframe/src/Iframe.js]: I understand avoiding visually cluttering the display. But... could we have this in `input` ? :-)
A number of genres benefit from pointer input (lemmings, duck hunt, simcity...)

Comment (gabrielflorit) [reply src/iframe/src/Iframe.js]: Let me think about it.

**Comment by Kethku** [comment]
Completely understandable. I made the PR on a take it or leave it basis.

**Pull request closed without merging**